.travis.yml

```diff
@@ -32,20 +32,23 @@ jobs:
 
   include:
     # runs linting and tests with current locked deps
-
     - stage: "Tests"
       name: "Tests"
       script:
-        - yarn lint:hbs
-        - yarn lint:js
-        - yarn test
+        - yarn lint
+        - yarn test:ember
+
+    - stage: "Additional Tests"
+      name: "Floating Dependencies"
+      install:
+        - yarn install --no-lockfile
+      script:
+        - yarn test:ember
```

Comment:
Why this new block? Because it exists when creating an add-on in v3.24:

https://github.com/ember-cli/ember-addon-output/blob/07fe28b1c23a909ce9dabd04d482003c56614429/.travis.yml#L41-L46